containers/datalab/Dockerfile.in

```diff
@@ -36,5 +36,7 @@ ENV EXPERIMENTAL_KERNEL_GATEWAY_URL=""
 # Startup
 ENV DATALAB_VERSION _version_
 ENV DATALAB_COMMIT _commit_
-ENTRYPOINT [ "/datalab/run.sh" ]
+ENV DEPLOY_VM "false"
```

Comment:
> This looks like a "hidden" feature that will almost never be used and exercised. IMO it is better to remove it so we don't need to maintain it.

That's the plan, but I don't want to bundle that in to the same pull request as the rest of the changes here.

In particular, I want to gain more confidence in the default user flow that this change introduces before ripping out the previous user flow.

> Maybe we can also provide a demo deploy.sh to help user create the VM.

I think any script we supply should be in a Docker image so that we can ensure the required dependencies are present and that the script works cross platform.

That's essentially all the "-e DEPLOY_VM=true" functionality is (except for being triggered off of an environment variable rather than a separate entry point).